packages/react/src/chat/BaseChat/BaseChat.tsx

```diff
@@ -92,6 +93,7 @@ export function BaseChat(props: BaseChatProps): JSX.Element | null {
     onMessageUpdated,
     inputDisabled,
     onError,
+    excludeHeader = false,
```

Comment:
Does this need default value?  If `excludeHeader` is undefined, everything would still work, yeah?